pupa/cli/commands/update.py

```diff
@@ -59,6 +59,46 @@ def save_report(report, jurisdiction):
             )
 
 
+@transaction.atomic
+def save_report(report, jurisdiction):
+    from pupa.models import RunPlan
+    plan = RunPlan.objects.filter(jurisdiction=jurisdiction).latest('id')
```

Comment:
We have the real plan before we invoke this method; I wonder if we can find a way to pass the run plan we have above down here through magic. Thoughts?